src/logger.ts

```diff
@@ -25,16 +25,21 @@ const teams = {
 		background: '#000000',
 		font: '#ff7300',
 	},
+	cmp: {
+		background: '#FF1493',
+		font: '#333',
```

Comment:
Whilst we’re relying on WebAIM’s API, we have to use 6-digit hexes.
```suggestion
		font: '#333333',
```